user/languages/julia.md

```diff
@@ -30,8 +30,9 @@ versions, use the `julia:` key in your `.travis.yml` file, for example:
 language: julia
 julia:
   - nightly
-  - 0.5
-  - 0.5.2
+  - release
```

Comment:
this shouldn't be mentioned in docs or encouraged - support for it should be deleted from the code, really

Reply:
why so?

Reply:
Same reason it's been discouraged for years - "latest release" changes what's being tested against over time, but the information about supported versions of Julia for a package is contained in REQUIRE or Project.toml. Those versions are different and will get out of sync over time. It'd be great if the versions to test against could be auto-detected from the same source of information that Pkg uses and populate the matrix from that, but that's non-trivial. `release` is a legacy, long-obsolete holdover that made porting ancient `.travis.yml` files that were referencing the old juliareleases ppa a direct translation when `language: julia` was brand new. The ppa never had the direct flexibility to keep testing against all julia versions a package claims to support, the modern recommended way that declares explicit versions does.

Reply:
I hadn't seen any discouragement before, but that seems reasonable. In that case, we should probably throw a warning, and change the defaults.

Reply:
The defaults in pkgdev were changed in https://github.com/JuliaLang/PkgDev.jl/pull/108 and I went and submitted PR's to any package that had still been using `release` around that time, and removed it from the docs here in https://github.com/travis-ci/docs-travis-ci-com/pull/1188. Without it being in the PkgDev default script any more, I suspect very few people have intentionally added it back since. Hopefully it's not used in PkgTemplates or whatever people are using nowadays.

Reply:
On a related note: do you think we should now allow a single digit? i.e. `1` will give you the latest `1.y.z` release?

Reply:
I'd have to check the code, but that might already work automatically if there's a julia-1-latest binary download put in the right place on s3